app/src/main/java/fr/free/nrw/commons/media/MediaDetailPagerFragment.java

```diff
@@ -332,13 +332,26 @@ private void updateBookmarkState(MenuItem item) {
 
     public void showImage(int i, boolean isWikipediaButtonDisplayed) {
         this.isWikipediaButtonDisplayed = isWikipediaButtonDisplayed;
-        Handler handler =  new Handler();
-        handler.postDelayed(() -> pager.setCurrentItem(i), 5);
+        settingViewPagerCurrentItem(i);
     }
 
     public void showImage(int i) {
-        Handler handler =  new Handler();
-        handler.postDelayed(() -> pager.setCurrentItem(i), 5);
+        settingViewPagerCurrentItem(i);
+    }
+    private void settingViewPagerCurrentItem(int i) {
```

Comment:
Since a few years our rule is that we need javadoc for every method addition. Even if self-explanatory. And yes, I know that most methods do not have javadoc yet, we are working on it :-)
Also, one empty line to separate it from the previous method.
Thanks!